operations/research-and-development/product/release-process/bug-fix-release.md

```diff
@@ -254,8 +254,11 @@ The final release is cut - RC cuts and bug fixes should be completed by this dat
             - Upgrade dependancies for webapp, server, and Redux
         - Week after release (for GitLab dev owner)
             - [Submit GitLab Omnibus RC install of Mattermost](https://mattermost.atlassian.net/browse/MM-9872)
-        - The week of code complete:
+        - The week before code complete:
             - [Add telemetry for new configuration settings in the release](https://mattermost.atlassian.net/browse/MM-24483)
+         - The week before code complete:
+            - If an existing ESR is going out of support next month, update the in-app prompts (mobile and desktop) to start detecting for the new minimum supported ESR version
```

Comment:
Will there be two tickets for Mobile and Apps?

Reply:
One for mobile, one for desktop